src/libs/ReportActionsUtils.ts

```diff
@@ -39,6 +39,8 @@ type MemberChangeMessageRoomReferenceElement = {
 
 type MemberChangeMessageElement = MessageTextElement | MemberChangeMessageUserMentionElement | MemberChangeMessageRoomReferenceElement;
 
+const policyChangeActionsSet = new Set<string>(Object.values(CONST.REPORT.ACTIONS.TYPE.POLICYCHANGELOG));
```

Comment:
NAB: I don't really think this needs to be a const, why don't we just use `new Set<string>(Object.values(CONST.REPORT.ACTIONS.TYPE.POLICYCHANGELOG))` directly?

Reply:
See this thread: https://github.com/Expensify/App/pull/35525#discussion_r1473615974.